src/main/java/org/folio/rest/impl/EholdingsUsageConsolidationImpl.java

```diff
@@ -23,39 +23,39 @@
 import org.folio.rest.jaxrs.model.UCSettingsPatchRequest;
 import org.folio.rest.jaxrs.model.UCSettingsPostRequest;
 import org.folio.rest.jaxrs.resource.EholdingsKbCredentialsIdUc;
+import org.folio.rest.jaxrs.resource.EholdingsUc;
 import org.folio.rest.util.ErrorHandler;
 import org.folio.service.uc.UCSettingsService;
 import org.folio.service.uc.UcAuthenticationException;
 import org.folio.spring.SpringContextUtil;
 
-public class EholdingsKbCredentialsUCImpl implements EholdingsKbCredentialsIdUc {
+public class EholdingsUsageConsolidationImpl implements EholdingsKbCredentialsIdUc, EholdingsUc {
 
   @Autowired
   @Qualifier("securedUCSettingsService")
   private UCSettingsService settingsService;
   @Autowired
   private ErrorHandler errorHandler;
 
-  public EholdingsKbCredentialsUCImpl() {
+  public EholdingsUsageConsolidationImpl() {
     SpringContextUtil.autowireDependencies(this, Vertx.currentContext());
   }
 
   @Override
-  public void getEholdingsKbCredentialsUcById(String credentialsId, Map<String, String> okapiHeaders,
-                                              Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
-    settingsService.fetchByCredentialsId(credentialsId, okapiHeaders)
+  public void getEholdingsUc(Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler,
+                             Context vertxContext) {
+    settingsService.fetchByUser(okapiHeaders)
       .thenAccept(ucSettings -> asyncResultHandler.handle(succeededFuture(
         GetEholdingsKbCredentialsUcByIdResponse.respond200WithApplicationVndApiJson(ucSettings))))
```

Comment:
Different response class. should be GetEholdingsUcResponse

Reply:
Fixed